### PR TITLE
FALCON-1818 : Minor doc update for tar package locations after FALCON - 1751

### PR DESCRIPTION
**Pull request by deepakbarr**

### CODE REVIEW

**Comment by bvellanki** [comment]
+1

File: CHANGES.txt [hunk 1]
```diff
@@ -84,6 +84,8 @@ Proposed Release Version: 0.9
     FALCON-1213 Base framework of the native scheduler(Pallavi Rao)
 
   IMPROVEMENTS
+    FALCON-1707 Minor doc update for tar package locations after FALCON-1751 (Deepak Barr)
```
Comment (pallavi-rao) [inline CHANGES.txt]: This change won't go into 0.9. So, can move this change up, under trunk section, please?

**Comment by pallavi-rao** [comment]
+1, other than the minor change log nit

**Comment by ajayyadava** [comment]
:+1: Will commit shortly.

**Comment by sandeepSamudrala** [comment]
:+1:

**Pull request closed without merging**